projects/Mallard/src/helpers/push-download-failsafe.ts

```diff
@@ -0,0 +1,39 @@
+import BackgroundFetch from 'react-native-background-fetch'
+import { pushTracking } from './push-tracking'
+import { clearAndDownloadIssue } from './clear-download-issue'
+
+const pushDownloadFailsafe = () => {
+    BackgroundFetch.configure(
+        {
+            minimumFetchInterval: 120, // Every 2 hours
+            stopOnTerminate: false,
+            startOnBoot: true,
+        },
+        async () => {
+            await pushTracking('backgroundFetch', 'started')
+            await clearAndDownloadIssue()
+            await pushTracking('backgroundFetch', 'ended')
+            BackgroundFetch.finish(BackgroundFetch.FETCH_RESULT_NEW_DATA)
+        },
+        error => {
+            pushTracking('backgroundFetchError', error.toString())
+        },
+    )
+
+    const ID = 'backgroundFetchStatus'
+    BackgroundFetch.status(status => {
+        switch (status) {
```

Comment:
Does this need a default case?

Reply:
Good question, i would say not. It gives a little extra information and we only really care about those who are enabled.